README.md

```diff
@@ -136,7 +103,22 @@ class Product {
 ```
 
 Afterwards you are able to use the created entity as if you were using a database.
-Let's assume we have used the value ```http://www.yourSite.com/api/products``` for the product entity's ```@Table``` annotation.
+
+By using this setting the driver is doing a lot of stuff by itself:
```

Comment:
die unterscheidung zwischen basic und adavanced sollte auch in nem absatz vor den beiden themen erklärt werden, also nach dem motto:

```
Usage

Et gibt 2 möglichkeiten: Einfach, wo fast lles automatisch is, wenn prerequisites eingehalten werden, Advanced, wo viel konfigurierbar ist, dat wird jetz Erklärt:

Einfach
...
Advanced....
```